src/ActionList.jsx

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
+import dragula from 'react-dragula';
```

Comment:
Wouldn't `DragSource.canDrag` still implies extra complexity when not needed? In Redux Extension `3.0`, except actions, we'll have events/effects, and we don't want features like this one to affect the perf there, so I guess having 2 different components would be better. What concerns me the most is the second part ("use local state to drop the element before getting the data from the client part, which takes time in case of the extension"). Basically, we'll have to duplicate the functionality from https://github.com/zalmoxisus/redux-devtools-instrument/pull/13/commits/13200dde11a07cc4ec30ea827903eed73bf0b78a via localState to drop the action, as it's not out of the box like in `react-dragula`.